docs/orchestration/concepts/kv_store.md

```diff
@@ -0,0 +1,123 @@
+# KV Store
```

Comment:
Am I correct that this will be a cloud only feature?  If so, can we add the Cloud badge?

Reply:
Yeah that sounds proper

Reply:
good call, updated